lighty-examples/lighty-controller-springboot-netconf/src/main/java/io/lighty/core/controller/springboot/config/LightyConfiguration.java

```diff
@@ -111,13 +115,26 @@ void shutdown() {
     void closeLightyModule(final LightyModule lightyModule) {
         if (lightyModule != null) {
             final Stopwatch stopwatch = Stopwatch.createStarted();
+            LOG.info("Lighty module {} shutting down ...", lightyModule);
             try {
-                LOG.info("Lighty module {} shutting down ...", lightyModule);
-                lightyModule.shutdown();
+                final ListenableFuture<Boolean> future = lightyModule.shutdown();
+                Futures.addCallback(future, new FutureCallback<>() {
+                    @Override
+                    public void onSuccess(Boolean result) {
+                        if (result) {
+                            LOG.info("Lighty module {} stopped in {}", lightyModule, stopwatch.stop());
+                        } else {
+                            LOG.info("Lighty module {} failed to stop", lightyModule);
```

Comment:
`LOG.info("Lighty module {} failed to stop after {}", lightyModule, stopwatch.stop());`